### PR TITLE
Fixed undefined value returned by getEdgeDirectory function in Windows Server

### PR DESCRIPTION
**Pull request by angulito**
@ifisidro and me are developing a CI system into a Windows server machine, and Windows Server 2016 does not have Microsoft Edge installed by default, so the function getEdgeDirectory allocated into _windows.js_ file returns undefined and after that, path.join function fails.

### CODE REVIEW

**Comment by chasenlehara** [comment]
Hi @AnguloHerrera, thanks for the PR! There are a couple of listing errors, you can see them in the Travis build: https://travis-ci.org/bitovi/launchpad/jobs/333708819

**Comment by AnguloHerrera** [comment]
Right! I didn't see it, I will take a look and I will fix them. Thanks you!

**Comment by AnguloHerrera** [comment]
@chasenlehara Maybe could you take a look to the latest TravisCI error? I have seen that you have some builds  with the same error:
"_No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received_"
Thanks a lot.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "launchpad",
   "description": "You can launch browsers! From NodeJS! Local ones! Remote ones! Browserstack ones!",
-  "version": "0.7.0",
+  "version": "0.7.1",
```
Comment (chasenlehara) [inline package.json]: No need to manually bump this number, we have a release script: https://github.com/bitovi/launchpad/blob/91d7782e48395af9367246bd8da148a5a84cd155/package.json#L36

**Comment by AnguloHerrera** [comment]
@chasenlehara Reverted, although TravisCI still has the same error...

**Comment by AnguloHerrera** [comment]
Hi @chasenlehara, I have discovered that the Firefox browser test into Linux machines does not response to the server.once('request', ...). It means that this event never is called and the test command gives timeout. You can check it with log traces into TravisCI or removing the firefox platform for Unix platforms (only for testing it). Maybe you know how it could fix. Thanks!!

**Comment by AnguloHerrera** [comment]
Hello @chasenlehara, any update? I don't know exactly how to solve the Firefox test problem on Linux machines, and I can not test it because I don't use Linux.

**Comment by Sanyusha** [comment]
Hello,
Is "undefined value fix" going to be merged?
Thank you!

**Comment by justinbmeyer** [comment]
We are having trouble with travis running the tests.  @cherifGsoul Any updates?

**Comment by cherifGsoul** [comment]
@justinbmeyer I'm planning to work on it on Friday (After tomorrow).

**Comment by angulito** [comment]
thx @cherifGsoul

**Comment by Sanyusha** [comment]
Hi, any updates on this?
Thank you.

File: lib/local/index.js [hunk 2]
```diff
@@ -56,8 +56,28 @@ module.exports = function (settings, callback) {
     }).nodeify(callback);
   };
 
+  api.customBrowsers = function(custombrowsers, callback) {
```
[the remark this answers is not in the document]
Comment (angulito) [reply lib/local/index.js]: if I remember well, add the possibility to just use some of the custom browsers and not all of the OS

**Pull request closed without merging**